### DISCUSSION TITLE
Spotify Integration

### DISCUSSION THREAD

**Discussion by ryanalexander** [stoatchat/discussions#485] [Feature Suggestions]
**Suggested by bailey2048 (01F5H3CHYBXQNA16JJPRQJ4CME) in Suggestions on Revolt server**

Now Playing support similar to Discord's integration, and quite potentially even a small client tooltip similar to what's pictured that shows the currently playing track to you.

**Comment by tjaynl** [comment]
maybe instead of spotify i would suggest nuclear.js (https://github.com/nukeop/nuclear) its free and open source and as far as i know doesn't far your private data.

**Comment by Rexogamer** [comment]
I think the thing with Spotify is that, well, most people use it - if you want to open a separate issue for Nuclear support you are more than welcome to but this issue is focused on Spotify

**Comment by tjaynl** [comment]
My suggestion was more based on the privacy issues that would come with spotify so thats why i suggested an alternative instead.

**Reply by MasicoreLord** [reply]
Maybe a disclaimer about the potential risks, otherwise it'd be a nice bonus to be able to have integration with widely used services like Spotify as well as any of the niche ones.
